src/test/java/io/appium/java_client/ios/XCUIDriverTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.java_client.ios;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import io.appium.java_client.remote.HideKeyboardStrategy;
+
+import org.junit.Test;
+
+import io.appium.java_client.MobileBy;
+import org.junit.*;
+import org.openqa.selenium.Point;
+import org.openqa.selenium.ScreenOrientation;
+import org.openqa.selenium.html5.Location;
+	
+public class XCUIDriverTest extends BaseIOSTest {
```

Comment:
Please add remove _extends BaseIOSTest _ and **@BeforeClass**/**@BeforeTest** and **@AfterClass**/**@After** methods. They should instantiate IOSDriver with specific capabilities, I think. May be it needs some specific server flags. 

This test won't work. It is using regular iOS UI Automation.